src/webgpu/api/validation/render_pass/resolve.spec.ts

```diff
@@ -129,17 +144,25 @@ Test various validation behaviors when a resolveTarget is provided.
             usage: resolveTargetUsage,
           });
 
-          renderPassColorAttachmentDescriptors.push({
-            view: resolveSourceColorAttachment.createView(),
-            loadOp: 'load',
-            storeOp: 'discard',
-            resolveTarget: resolveTarget.createView({
+          let resolveTargetView: GPUTextureView | undefined;
+
+          const shouldError =
+            resolveTargetViewMipCount === 0 || resolveTargetViewArrayLayerCount === 0;
+          t.expectValidationError(() => {
+            resolveTargetView = resolveTarget.createView({
```

Comment:
When resolveTargetViewMipCount or resolveTargetViewArrayLayerCount is 0, createView generates a validation error. So I make createView expect a error when they are 0.